program/index.html

```diff
@@ -52,6 +61,7 @@ <h2 class='color-purple2 spartan'>Program</h2>
         </header>
 
         <!-- Friday -->
+        <h2>Friday</h2>
```

Comment:
Can you remove this title? It's repetitive and breaking the layout

![image](https://cloud.githubusercontent.com/assets/2894119/16673005/5bfaddae-4478-11e6-888b-44a074440872.png)